pywhat/Data/regex.json

```diff
@@ -703,19 +703,18 @@
        ]
     },
     {
-       "Name": "American Social Security Number",
-       "Regex": "^[0-9]{3}-[0-9]{2}-[0-9]{4}$",
-       "plural_name": false,
-       "Description": "An [#CAE4F1][link=https://en.wikipedia.org/wiki/Social_Security_number]American Identification Number[/link][/#CAE4F1]",
-       "Rarity": 0.2,
-       "URL": null,
-       "Tags": [
-          "Credentials",
-          "Password",
-          "Username",
-          "SSN"

```

Comment:
Please add `SSN` tag back. It came from another PR, so that's why it probably slipped through :)

Reply:
Done.